requirements.txt

```diff
@@ -0,0 +1 @@
+awscli==1.7.25
```

Comment:
Hmmm.. This is getting complicated.  What if we are neither ruby or Python?

On Sun, May 3, 2015 at 5:45 AM, Ibrahim Rashid <notifications@github.com>
wrote:

>> @@ -0,0 +1 @@
>> +awscli==1.7.25
> @kbaum, It installs `awscli` when you push you code to heroku if you are in python application. 
> for ruby we can include a gem file with with this content `gem 'aws-cli', '~> 0.0.1.pre4'` . Providing both the `requirement.txt` and the `gemfile` will be ok. 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/kbaum/heroku-database-backups/pull/5/files#r29554548